packages/table/tests/SortableHeaderCell.test.tsx

```diff
@@ -6,7 +6,6 @@ import { createSerializer } from "jest-emotion";
 import { shallow } from "enzyme";
 import toJson from "enzyme-to-json";
 import HeaderCell from "../components/HeaderCell";
-import Clickable from "../../clickable/components/clickable";
```

Comment:
This was breaking tests, even in master. Maybe related to some recent package upgrades?